opentracing_utils/libs/_requests.py

```diff
@@ -46,7 +46,20 @@ def trace_requests(default_tags=None, set_error_tag=True, mask_url_query=True,
     :param ignore_url_patterns: Ignore tracing for any URL's that match entries in this list
     :type ignore_url_patterns: list
     """
-    @trace(pass_span=True, tags=default_tags)
+    def skip_span_matcher(*args, **kwargs):
```

Comment:
I thought so as well, but all of the requests library test fail if we don't pass `*args`. I found that in these cases the `*args` are actually:
```
args = (<requests.adapters.HTTPAdapter object at 0x102d854a8>, <PreparedRequest [GET]>)
```

I suppose it is because when using `trace_requests(ignore_url_pattern=[...])` then `skip_span_matcher` will be called for all requests and seems like the argument list may differ.

Reply:
hmm, I think the first arg is ``self``, so ``(self, request, **kwargs)`` should work I guess.